invenio_rdm_records/services/permissions.py

```diff
@@ -84,6 +84,8 @@ class RDMRecordPermissionPolicy(RecordPermissionPolicy):
     can_new_version = can_curate
     # Allow publishing a new record or changes to an existing record.
     can_publish = can_curate
+    # Allow lifting a record or draft.
+    can_lift = can_manage
```

Comment:
what about `can_lift_embargo`, I understand is longer but otherwise might not be clear that it means embargo